### PR TITLE
Update Bug.md

### PR DESCRIPTION
**Pull request by chryzcodez**
#### What does this PR do?

Updates the name of the community and activated the link directing people to the community

#### What is the relevant issue link?
https://github.com/EddieHubCommunity/support/issues/881

### CODE REVIEW

**Review by SashoStoichkov — APPROVED** [review]
Looks good 👍🏻

File: .github/ISSUE_TEMPLATE/bug.md [hunk 1]
```diff
@@ -33,4 +33,4 @@ Add any other context about the problem here.
 
 You can also join the discord community [here](https://discord.com/invite/jZQs6Wu)
 
-Feel free to check out other cool repositories of Eddie Jaoude Community [here](https://github.com/EddieJaoudeCommunity)
+Feel free to check out other cool repositories of EddieHub Community [here](https://github.com/EddieHubCommunity)
```
Comment (SashoStoichkov) [inline .github/ISSUE_TEMPLATE/bug.md:36]: Maybe here should be only `EddieHub` 🤔

**Review by SashoStoichkov — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug.md (hunk 1 above)
Comment (chryzcodez) [reply .github/ISSUE_TEMPLATE/bug.md:36]: It name of the community wasn't changed to EddieHub but EddieHub community, that was why I did like that.

Comment (SashoStoichkov) [reply .github/ISSUE_TEMPLATE/bug.md:36]: @eddiejaoude, which of the two you think is better?

**Review by BOLT04 — APPROVED** [review]
LGTM 👍, thanks for updating the issue template @chryzcodez

File: .github/ISSUE_TEMPLATE/bug.md (hunk 1 above)
Comment (BOLT04) [reply .github/ISSUE_TEMPLATE/bug.md:36]: I've approved the PR @chryzcodez, but let's wait for @eddiejaoude feedback on this topic before merging😃

**Comment by SashoStoichkov** [comment]
Thanks @BOLT04 👍🏻

File: .github/ISSUE_TEMPLATE/bug.md (hunk 1 above)
Comment (eddiejaoude) [reply .github/ISSUE_TEMPLATE/bug.md:36]: Thanks all 👍 . I think `EddieHub` might be cleaner. I will merge this, as it still fixes the link, but if a new PR could be created to just use `EddieHub`

**Review by eddiejaoude — APPROVED** [review]
thanks 👍

**Pull request merged**